mailbox/api/src/main/java/org/apache/james/mailbox/MessageManager.java

```diff
@@ -118,7 +117,7 @@
      * @throws MailboxException
      *             if anything went wrong
      */
-    Iterator<MessageUid> expunge(MessageRange set, MailboxSession mailboxSession) throws MailboxException;
+    Stream<MessageUid> expunge(MessageRange set, MailboxSession mailboxSession) throws MailboxException;
```

Comment:
also here?

Reply:
What do you mean?

Reply:
it was about adding the comment on stream.close